### ISSUE TITLE
What does VERSION under session headers actually achieve?

### ISSUE THREAD

**Issue by RRMXkun** [ig-python/ig-markets-api-python-library#159]
I've looking to fix an issue with switch_account. Issue being VERSION is not specified so it picks up whichever was last set by another method, so it's roulette as to whether it will work or not. Then I noticed in rest.py that sort of thing all over the place:

e.g. _create_open_position()_:
```
 response = self._req(action, endpoint, params, session, version="2")
        if "Version" in self.crud_session.HEADERS["LOGGED_IN"]:
            del self.crud_session.HEADERS["LOGGED_IN"]["VERSION"]
```
Sometimes we specify version in the call itself e.g. _fetch_working_orders()_:
`response = self._req(action, endpoint, params, session, version="1")`

And sometimes we even set the session's version number on the fly e.g. _fetch_transaction_history()_:
 ```
  self.crud_session.HEADERS["LOGGED_IN"]["VERSION"] = "2"
        response = self._req(action, endpoint, params, session)
        del self.crud_session.HEADERS["LOGGED_IN"]["VERSION"]
```
It's very inconsistent all over.

Question: why are we even bothering with this field in the headers? Since VERSION will vary for each method, why not just define it at the time the call is made? I feel like I'm missing something here?

If not then suggest we:

1.  Remove VERSION from HEADERS["LOGGED_IN"]
2.  Re-organise every method in rest.py that require VERSION so they are all consistent e.g.:

  ```
  def delete_working_order(self, deal_id, session=None):
        """Deletes an OTC working order"""
        VERSION = "2"
[...]
        response = self._req(action, endpoint, params, session, VERSION)
```
Happy to do this, but didn't want to go ahead and do it before I am confident there isn't another reason to have VERSION in the session headers. Implementing my suggestion will also fix bugs with certain methods such as _delete_working_order()_, _switch_account()_ etc which do not currently specify a version number and will therefore most likely fail.

**Comment by bug-or-feature** [comment]
I also don't understand why VERSION is stored, and I agree we should be more consistent. I can't see any instance where the VERSION value is read, only that it is removed. Its hard to see what the original author's intention was

**Comment by bug-or-feature** [comment]
Closed by #161
